src/sentry/api/endpoints/organization_auditlogs.py

```diff
@@ -46,10 +46,14 @@ def get(self, request: Request, organization) -> Response:
             else:
                 queryset = queryset.filter(event=query["event"])
 
-        return self.paginate(
+        response = self.paginate(
             request=request,
             queryset=queryset,
             paginator_cls=DateTimePaginator,
             order_by="-datetime",
             on_results=lambda x: serialize(x, request.user),
         )
+        # TODO: Cleanup after frontend is fully moved to version 2
+        if "version" in request.GET:
+            response.data = {"rows": response.data, "options": audit_log.get_api_names()}
```

Comment:
It'll be nice to have a test for the version 2 response. 👀

Reply:
Agreed, we'll need tests before we can merge.